### DISCUSSION TITLE
【2205750】Waifu Fighter -Family Friendly 女拳主义F-ist！一般向版（本体）

### DISCUSSION THREAD

**Discussion by miluoqiu** [pjy612/SteamManifestCache#342] [Share]
appid : `2205750`

keys:

```
2205751;a67785ee7b87b686c910bea65d17e0fd28bfb807ad2a6619fb41f53933547c9b
```

code：AE8B3FF49CAEB8B5
email：sglshou@qq.com

[女拳鬥士F-ist.zip]（ 密钥+清单）(https://github.com/pjy612/SteamManifestCache/files/13872009/F-ist.zip)

**Comment by pjy612** [comment]
请修正密钥格式

**Reply by miluoqiu** [reply]
这样应该可以了吧？

**Reply by pjy612** [reply]
目前看着可以
